client/resource_group/controller/controller.go

```diff
@@ -491,31 +506,14 @@ func (gc *groupCostController) initRunState() {
 	}
 }
 
-func (gc *groupCostController) updateRunState(ctx context.Context) {
+func (gc *groupCostController) updateRunState() {
 	newTime := time.Now()
-	deltaConsumption := &rmpb.Consumption{}
+	gc.mu.Lock()
 	for _, calc := range gc.calculators {
-		calc.Trickle(ctx, deltaConsumption)
+		calc.Trickle(gc.mu.consumption, gc.mainCfg.IsSingleGroupByKeyspace)
 	}
-	gc.mu.Lock()
-	add(gc.mu.consumption, deltaConsumption)
 	*gc.run.consumption = *gc.mu.consumption
 	gc.mu.Unlock()
-	// remove tokens
```

Comment:
If IsSingleGroupByKeyspace is false, do we need to add SQL CPU to RU consumption as well?

Reply:
I mean we can keep this code.

Reply:
I think this time pr is mainly for the IsSingleGroupByKeyspace case.
maybe better add this code when implementing multiple groups.